### PR TITLE
feat: SSO MFA - add missing SAML MFA fields

### PR DESCRIPTION
**Pull request by Joerger**
Follow up to https://github.com/gravitational/teleport/pull/46703 adding missing fields needed for the SSO MFA flow.

### CODE REVIEW

File: integrations/terraform/Makefile [hunk 1]
```diff
@@ -46,7 +46,7 @@ $(BUILDDIR)/terraform-provider-teleport_%: terraform-provider-teleport-v$(VERSIO
 
 CUSTOM_IMPORTS_TMP_DIR ?= /tmp/protoc-gen-terraform/custom-imports
 # This version must match the version installed by .github/workflows/lint.yaml
-PROTOC_GEN_TERRAFORM_VERSION ?= v2.2.0
+PROTOC_GEN_TERRAFORM_VERSION ?= v3.0.0
```
Comment (Joerger) [inline integrations/terraform/Makefile:49]: @hugoShaka I wasn't able to download `github.com/gravitational/protoc-gen-terraform@v2.2.0` so I updated this to v3.0.0. Is this correct or is there a way for me to get v2.2.0 still?

Comment (Joerger) [reply integrations/terraform/Makefile:49]: I think this just got fixed on master, sorry for the ping.

File: integrations/terraform/tfschema/accesslist/v1/accesslist_terraform.go [hunk 2]
```diff
@@ -60,7 +61,11 @@ func GenSchemaAccessList(ctx context.Context) (github_com_hashicorp_terraform_pl
 							Optional:    true,
 							Type:        github_com_hashicorp_terraform_plugin_framework_types.StringType,
 						},
-						"expires": GenSchemaTimestamp(ctx),
+						"expires": GenSchemaTimestamp(ctx, github_com_hashicorp_terraform_plugin_framework_tfsdk.Attribute{
+							Description: "expires is a global expiry time header can be set on any resource in the system.",
+							Optional:    true,
+							Validators:  []github_com_hashicorp_terraform_plugin_framework_tfsdk.AttributeValidator{github_com_gravitational_teleport_integrations_terraform_tfschema.MustTimeBeInFuture()},
+						}),
```
Comment (hugoShaka) [inline integrations/terraform/tfschema/accesslist/v1/accesslist_terraform.go:68]: I think you are using a too recent of the tfschema protobuf generator. We recently made the change to pass the attribute data to the schema generation function and have not converted the provider code yet.

File: integrations/terraform/Makefile (hunk 1 above)
Comment (hugoShaka) [reply integrations/terraform/Makefile:49]: Yeah you should re-render with the old version. The TF codebase is not ready for 3.0.0, the code from the PR will not compile

**Review by rosstimothy — APPROVED** [review]

**Comment by Joerger** [comment]
@gzdunek @smallinsky friendly ping to review

**Review by gzdunek — APPROVED** [review]

**Pull request merged**